test/queries/languages.test.js

```diff
@@ -13,9 +13,8 @@ test('test Languages query', (t) => {
   ];
   dbBuild()
     .then(() => getLanguages())
-    .then((res) => res.rows)
-    .then((result) => {
-      t.deepEqual(result, expected, 'The table should be languages');
+    .then(({ rows }) => {
+      t.deepEqual(rows, expected, 'The table should be languages');
```

Comment:
unrelated file

Reply:
Yes, but I find that this is better to edit it in the same pull request 
I will add it in its branch